Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
I didn't know we accessed it from Foundation; I thought we went through _CFRuntimeBridgeTypeToClass! Cool.

---
In reply to: [79025442](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79025442) [](ancestors = 79025442,79022634,78866010,78862337)